### ISSUE TITLE
No Temperatures in App

### ISSUE THREAD

**Issue by JoeSem** [mskg/homey-heating#200]
Ive installet Homey heating Shedule in Version 2.0.7 at my Homey Pro 2023 and tryed to make my 1st heating schedule.
But there are no actual temperatures in the schedule avaliable. 
What is my mistake? Using Aeontec Multisensor 6 and 7 and have them added to my rooms.
Homey show all themperatures in the Climate section.

**Comment by khandanpour** [comment]
Yes I have the sae Problem. And in settings and devices thera are no devices aviable. And I can't ser up actual temperature.
